okio/build.gradle.kts

```diff
@@ -210,6 +213,20 @@ kotlin {
       }
     }
   }
+
+  targets.withType<KotlinNativeTargetWithTests<*>> {
+    binaries {
+      // Configure a separate test where code runs in background
+      test("background", setOf(NativeBuildType.DEBUG)) {
+        freeCompilerArgs += listOf("-e", "okio.mainBackground")
```

Comment:
I'm not sure a custom entrypoint is required and instead you can use the `-trw` flag with the regular one.

Reply:
Indeed, thanks for the suggestion.

Reply:
What does `trw` stand for?

Reply:
-generate-worker-test-runner (-trw)﻿
Produce an application for running unit tests in a worker thread.

test run in worker thread.

Reply:
https://kotlinlang.org/docs/compiler-reference.html#generate-test-runner-tr

Reply:
TIL. Thank you!